### DISCUSSION TITLE
Getting an error while Inspecting the extension

### DISCUSSION THREAD

**Discussion by Inzamamdev** [open-sauced-craftwork/browser-extension#25] [Q&A]
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

**Comment by nickytonline** [comment]
Can you provide more context to the issue? Sharing your repository with the problematic code would also be helpful.

**Comment by Inzamamdev** [comment]
Can you help me with the solution or guidance I have made the commit
